cmd/kata-manager/kata-manager.sh

```diff
@@ -0,0 +1,318 @@
+
+	config_checks
+
+	sudo sed -i "s!^#*image *=.*\$!image = \"$file\"!g" "$config_file"
+}
+
+enable_initrd()
+{
+	local file="$1"
+
+	config_checks
+
+	sudo sed -i "s!^#*initrd *=.*\$!initrd = \"$file\"!g" "$config_file"
+}
+
+cmd_enable_full_debug()
+{
+	info "enabling debug"
+
+	config_checks
+
+	sudo sed -i -e 's/^# *\(enable_debug\).*=.*$/\1 = true/g' "$config_file"
+	sudo sed -i -e "s/^kernel_params = \"\(.*\)\"/kernel_params = \"\1 ${agent_debug}\"/g" "$config_file"
+}
+
+cmd_disable_all_debug()
+{
+	info "disabling debug"
+
+	config_checks
+
+	sudo sed -i -e 's/^\(enable_debug.*=.*$\)/# \1/g' "$config_file"
+	sudo sed -i -e "s/^\(kernel_params = \".*\)${agent_debug}\(.*\"\)/\1 \2/g" "$config_file"
+}
+
+cmd_configure_image()
+{
+	local file="$1"
+
+	check_file "$file"
+
+	info "installing '$file' as image"
+
+	enable_image "$file"
+	disable_initrd
+}
+
+cmd_configure_initrd()
+{
+	local file="$1"
+
+	check_file "$file"
+
+	info "installing '$file' as initrd"
+
+	enable_initrd "$file"
+	disable_image
+}
+
+# Install the packaged version of Kata by executing the commands
+# specified in the installation guide document.
+cmd_install_packages()
+{
+	[ -z "${GOPATH}" ] && die "need GOPATH"
+
+	source /etc/os-release
+
+	local doc_repo_url="https://${doc_repo}"
+
+	local repo_dir="${GOPATH}/src/${doc_repo}"
+
+	info "installing packages"
+
+	[ ! -d "${repo_dir}" ] && (cd "$(dirname ${repo_dir})" && git clone "$doc_repo_url")
+
+	local file="${ID}-installation-guide.md"
+
+	local doc="${GOPATH}/src/${doc_repo}/install/${file}"
+	[ ! -e "$doc" ] && die "no install document for distro $distro"
+
+	local -r doc_script="kata-doc-to-script.sh"
+
+	local tool="${GOPATH}/src/${test_repo}/.ci/${doc_script}"
+	[ ! -e "${tool}" ] && die "cannot find script $doc_script"
+
+	local install_script=$(mktemp)
+
+	# create the script
+	"${tool}" "${doc}" "${install_script}"
+
+	# run the installation
+	bash "${install_script}"
+
+	# clean up
+	rm -f "${install_script}"
+}
+
+cmd_remove_packages()
+{
+	local packages_regex="^(kata|qemu-lite)-"
+	local packages
+
+	info "removing packages"
+
+	case "$distro" in
+		centos|fedora)
+			packages=$(rpm -qa|egrep "${packages_regex}" || true)
+			;;
+
+		ubuntu)
+			packages=$(dpkg-query -W -f='${Package}\n'|egrep "${packages_regex}" || true)
+			;;
+
+		*)
+			die "invalid distro: '$distro'"
+			;;
+	esac
+
+	[ -z "$packages" ] && { echo "packages not installed" && exit 1; }
+
+	case "$distro" in
+		centos) sudo yum -y remove $packages ;;
+		fedora) sudo dnf -y remove $packages ;;
+		ubuntu) sudo apt-get -y remove $packages ;;
+	esac
+}
+
+cmd_reset_config()
+{
+	# Cannot "undo" as the user has specified a custom config file
+	[ "$config_file" != "$local_config_file" ] && die "cannot reset config file '$config_file'"
+
+	sudo rm -f "$local_config_file"
+
+	local dir=$(dirname "$local_config_file")
+	sudo rmdir "$dir" 2>/dev/null || true
+}
+
+setup()
+{
+	source /etc/os-release
+
+	# lower-case distribution name
```

Comment:
this is no longer changing to lower-case here. modify or maybe delete comment?

Reply:
Comment zapped.